src/cctz.h

```diff
@@ -317,27 +317,27 @@ FloorSeconds(const time_point<seconds64>& tp) {
 }
 }  // namespace internal
 
+Breakdown BreakTime(const time_point<seconds64>&, const TimeZone&);
```

Comment:
I do not like that these functions become available globally, but otherwise I get ugly linker errors. Maybe someone will tell me better solution?